docs/internal-documentation/distrib-testing/Ubuntu-nodjs-build-environment.md

```diff
@@ -0,0 +1,53 @@
+# Ubuntu nodjs Build Environment
+
+The requirements to build the Zonemaster-GUI distribution zip file are nodejs
+and npm. Below are instructions to create such an build environment on Ubuntu.
+
+Nodjs and npn are available from the [Node.js] official website. Minimal version
+of Nodejs is 10.0 but install the latest LTS version available. The process has
+been tested on Ubuntu 18.04, which we use here.
+
+1. Make a clean installation of Ubuntu 18.04.
+
+> Are we root at this stage?
+
+2. Update the package database.
+
+   ```sh
+   apt-get update
+   ```
+
+3. Create a new user.
+
+   ```sh
+   adduser username
+   ```
+
+4. Add user to the sudoer list.
+
+   ```sh
+   visudo
+   ```
+
+   Copy the contents of the ‘root’ in 'visudo' and change ‘root’ to ‘username’.
```

Comment:
Usually this is automatically done during the installation process. You are prompt to create a new user. And this user will have sudo rights.

Maybe this is where the installation processes differ between Ubuntu and Debian.

Reply:
I remove the creation of user. Not needed.